app/buyers/views/buyers.py

```diff
@@ -362,6 +363,11 @@ def publish_brief(framework_slug, lot_slug, brief_id):
             question_and_answers['slug'] = section['id']
 
     unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+
+    if sections.get_section('set-how-long-your-requirements-will-be-live-for') and \
+            sections.get_section('set-how-long-your-requirements-will-be-live-for').questions[0].answer_required:
+            unanswered_required -= 1
+
     if request.method == 'POST':
         if unanswered_required > 0:
```

Comment:
I think it needs to be checked for get requests too due to the edge case that requirements length is the only unanswered required question. In this case the heading for the list of unanswered questions would be rendered, but there would be nothing in the list.